### ISSUE TITLE
Mscoco preprocessing flipping

### ISSUE THREAD

**Issue by achigeor** [bearpaw/pytorch-pose#41]
First of all thanks for the great repo!

In `pose/datasets/mscoco.py`, when the flipping happens, there is 
`pts = shufflelr(pts, width=img.size(2), dataset='mpii')`.

Does the flipping / mscoco training script work, or one should implement the matchedParts for mscoco?

Thanks!

**Comment by vicdu** [comment]
Do you plan to federate multi-datasets training hg-net?

**Comment by bearpaw** [comment]
@achigeor Sorry I didn't check the correctness of this part. Did you get correct results or not?
